### PR TITLE
Snow Globe!

### PR DESCRIPTION
**Pull request by icarusinthewings**
Thanks for making a PR! Please fill in the below:

**Name:**
Judah Brown

**Do you run a club?** 
Yes, Tulsa Hack Club

**What relevant experiences do you have to help you write this Jam?**
I've been doing web development for a while. Additionally, I have done some minimal work with ParticlesJS to know enough about how it works and some of the more advanced features that it has.

**Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam?**
Of Course!

### CODE REVIEW

**Review by faisalsayed10 — CHANGES_REQUESTED** [review]
Could you please answer the questions that are there regarding jams in your md file?

example:
image

**Comment by icarusinthewings** [comment]
@faisalsayed10 I answered them in the paragraph above. Was I supposed to just answer them within that list?

**Comment by icarusinthewings** [comment]
>  This jam is meant to give clubbers and newbies alike the chance to create something unique to them, while still preserving the "wow factor." The project is simple enough to follow along with, but open-ended enough to allow participants to create something akin to their knowledge level. Also, snow globes ❄️ are just fun! Participants will walk away with their own snow globe online! Club leaders can build along with the presentation and use their snow globe as an example. Personally, I want to build this jam to run as a program at my library to attract people to Tulsa Hack Club! I can't wait to see all the things people make!

**Comment by faisalsayed10** [comment]
Yeah I think it'd be better that way

**Comment by icarusinthewings** [comment]
Okay, got those added!

**Review by faisalsayed10 — APPROVED** [review]
LGTM

**Comment by icarusinthewings** [comment]
Yay!

**Review by linkai101 — APPROVED** [review]
ok i like it picasso

an optional addon could be adding sounds when you shake or move the globe 😁 

also just making sure, for devices without gyro capabilities, there's still touch controls right? id imagine it wouldn't be too big of an issue 👍

**Comment by icarusinthewings** [comment]
> ok i like it picasso
> 
> an optional addon could be adding sounds when you shake or move the globe 😁
> 
> also just making sure, for devices without gyro capabilities, there's still touch controls right? id imagine it wouldn't be too big of an issue +1

Of course!

File: jams/Snowglobe.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Details:
+_Outcome:_ Create your virtual snow globe that reflect who you are and that reacts to touch and motion
+
+_Requirements:_ Computer + IDE (Required, for coding. Replit is also acceptable), Internet Access (Required, for downloading assets), GitHub (Optional, for deployment), Mobile Device (Optional, for motion sensing)
+
+_Language used:_ HTML/JS/CSS
+
+_Customization opportunity:_ Change up the way your snow globe looks, what's precipitating, or even how the particles interact!
+
+_Platform Limitations:_ There is no platform limitation for the coding part of the project. However, due to an API limitation, the motion reaction is only available on devices with a gyroscope running one of the supported platforms. [See MDN](https://developer.mozilla.org/en-US/docs/Web/API/Gyroscope).
```
Comment (sahitid) [inline jams/Snowglobe.md:12]: is there an option to create the end project even without the gyroscope feature?

**Review by sahitid — COMMENTED** [review]
i personally love this jams! very excited for it

File: jams/Snowglobe.md (hunk 1 above)
Comment (icarusinthewings) [reply jams/Snowglobe.md:12]: Yes. You can interact with the snow globe using touch. The gyroscope is only needed for the motion reaction feature.

**Review by Arpan-206 — APPROVED** [review]
LGTM!

File: jams/Snowglobe.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Build Your Virtual "Own Globe": Synthesizing a reactive and custom snow globe!
+
+### Details:
+_Outcome:_ Create your virtual snow globe that reflect who you are and that reacts to touch and motion
+
+_Requirements:_ Computer + IDE (Required, for coding. Replit is also acceptable), Internet Access (Required, for downloading assets), GitHub (Optional, for deployment), Mobile Device (Optional, for motion sensing)
+
+_Language used:_ HTML/JS/CSS
+
+_Customization opportunity:_ Change up the way your snow globe looks, what's precipitating, or even how the particles interact!
```
Comment (sarthaktexas) [inline jams/Snowglobe.md:10]: can you add something for changing the actual particle? maybe it's not snow but like people? or sharks? etc. thanks!

File: jams/Snowglobe.md (hunk 1 above)
Comment (sarthaktexas) [inline jams/Snowglobe.md:12]: can you write this part into your proposal?

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
Overall love this, just add the things I commented and it's ready to be merged!

File: jams/Snowglobe.md (hunk 2 above)
Comment (icarusinthewings) [reply jams/Snowglobe.md:10]: Hey! That's what I meant when I said "change what's precipitating." Thanks!

File: jams/Snowglobe.md (hunk 1 above)
Comment (icarusinthewings) [reply jams/Snowglobe.md:12]: Sorry, I'm a bit confused by this comment.

Comment (sarthaktexas) [reply jams/Snowglobe.md:12]: it’s that you can do two different things but I think you did that, so resolve this if you agree

**Comment by SerenityUX** [comment]
WAHOOOO THIS IS AWESOME!

**Pull request merged**